### DISCUSSION TITLE
Money advice

### DISCUSSION THREAD

**Discussion by MirkosMangle** [NAIST-SE/AttractingContributors#1231] [General]
Where do you get money urgently if you don't have rich parents and relatives?

**Comment by hernessteve** [comment]
Platforms like Uber, Lyft, DoorDash, and Instacart offer quick income by driving, delivering food, or groceries. These services can provide immediate earnings, depending on how much you work.

**Comment by ellielo** [comment]
If you really need the money now, weigh the short-term benefits against the long-term growth potential. Generally, it's advisable to contribute enough to get the match and then reconsider early withdrawals. Check with the [regent bank customer service](https://www.pissedconsumer.com/company/regent-bank/customer-service.html) if they might give you more detailed information on that
